scripts/aks/bootstrap.sh

```diff
@@ -482,6 +482,14 @@ az aks create "${AKS_BASE_OPTIONS[@]}" "${AKS_OMNIA_OPTIONS[@]}" "${AKS_ENV_OPTI
 
 echo "Done."
 
+#######################################################################################
+### Lock cluster and network resources
+###
+if [ "$RADIX_ENVIRONMENT" = "prod" ]; then
+az lock create --lock-type CanNotDelete --name $CLUSTER_NAME-lock --resource-group clusters --resource-type Microsoft.ContainerService/managedClusters --resource $CLUSTER_NAME  2>&1 >/dev/null
+az lock create --lock-type CanNotDelete --name $VNET_NAME-lock --resource-group clusters --resource-type Microsoft.Network/virtualNetworks --resource $VNET_NAME  2>&1 >/dev/null
+fi
```

Comment:
Add indentation. When you want to suppress all output you can do `&>/dev/null` or `--output none`.
```suggestion
if [ "$RADIX_ENVIRONMENT" = "prod" ]; then
    az lock create --lock-type CanNotDelete --name ${CLUSTER_NAME}-lock --resource-group clusters --resource-type Microsoft.ContainerService/managedClusters --resource $CLUSTER_NAME  &>/dev/null
    az lock create --lock-type CanNotDelete --name ${VNET_NAME}-lock --resource-group clusters --resource-type Microsoft.Network/virtualNetworks --resource $VNET_NAME  &>/dev/null
fi
```